packages/algjulia-interop/jupyter_server.sh

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
 
-KERNEL_NAME="julia-1.11"
-#KERNEL_NAME="julia-ajaas-1.11"
+KERNEL="julia-ccl-interop-1.11"
```

Comment:
IMO, by default, we should not assume that the sys image is being used.

More important, though also more work, would be to make this script take command-line flags:

- `--kernel` (default: output of `julia -e 'print("julia-$(VERSION.major).$(VERSION.minor)")'`)
- `--mode`: one of `production`, `staging`, `dev` (default: `production`), resulting in `ORIGIN` be set appropriately
- `--origin`: direct specification of `ORIGIN` if `mode` is omitted

Then the user should never have to edit this script.